Game/Source/GDKTestGyms/OffloadingBenchmarkGymGameMode.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "OffloadingBenchmarkGymGameMode.h"
+
+#include "UObject/ConstructorHelpers.h"
+
+DEFINE_LOG_CATEGORY(LogOffloadingBenchmarkGymGameMode);
+
+AOffloadingBenchmarkGymGameMode::AOffloadingBenchmarkGymGameMode()
+{
+	ConstructorHelpers::FClassFinder<AActor> NPCBPClassFinder(TEXT("/Game/OffloadingBenchmark/OffloadingBenchmark_NPC_BP"));
+	NPCBPClass = NPCBPClassFinder.Class;
+
+	ConstructorHelpers::FClassFinder<AActor>  SimulatedPlayerBPClassFinder(TEXT("/Game/OffloadingBenchmark/OffloadingBenchmark_NPC_BP"));
+	SimulatedPlayerBPClass = SimulatedPlayerBPClassFinder.Class;
+}
+
+void AOffloadingBenchmarkGymGameMode::BuildExpectedObjectCounts()
+{
+	Super::BuildExpectedObjectCounts();
+
+	{
+		FExpectedActorCount ExpectedActorCount;
```

Comment:
Could we add an check for the dorp cube - it's most likely the one that breaks currently.

Reply:
Yes we can, my plan was to hand this to Danny once my this code is in. He can then add what is required for the benchmark gym. This was for the purpose of getting someone to use my "API" and also because Danny has a better understanding of the benchmark gym and can add all the required actors we need to keep track of. If there isn't much to add I can do it in this PR, just gimme a list of actors and numbers.